diff --git a/src/main/java/kr/motd/maven/os/Detector.java b/src/main/java/kr/motd/maven/os/Detector.java
index 8962f51..724e85a 100644
--- a/src/main/java/kr/motd/maven/os/Detector.java
+++ b/src/main/java/kr/motd/maven/os/Detector.java
@@ -200,6 +200,12 @@ public abstract class Detector {
         if ("ppc64le".equals(value)) {
             return "ppcle_64";
         }
+	if ("mips64".equals(value)) {
+            return "mips64";
+ 	}
+	if ("mips64el".equals(value)) {
+            return "mips64el";
+ 	}
         if ("s390".equals(value)) {
             return "s390_32";
         }

